.github/workflows/test-python.yml

```diff
@@ -36,19 +36,10 @@ jobs:
               exit 1
           fi
         working-directory: ./bindings/python
-      - name: Install Python dependencies
-        run: |
-          python -m pip install -U pip wheel
-          pip install 'cython>=0.29' 'pyarrow>=3' 'numpy>=1.16.6' 'pymongo>=3.11,<4' 'pandas' 'numpy >= 1.16.6'
-          python -c 'import pyarrow as pa; pa.create_library_symlinks()'
-        working-directory: ./bindings/python
```

Comment:
Don't we still need to install pyarrow and call create_library_symlinks?

Reply:
It's being done in `setup.py` to clean up the dev workflow

Reply:
Can you just revert this `Install Python dependencies` change? Yes, it duplicates the dependency versions but that's okay. It's better to have the dependency install separate from the pymongoarrow install. Makes it easier to debug failures.